lib/screens/v2/widget_instance/train_crowding.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Screens.V2.WidgetInstance.TrainCrowding do
+  @moduledoc """
+  A widget that displays the crowding on a train that is en route to the current station.
+  """
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Triptych
+  alias Screens.Predictions.Prediction
+
+  defstruct screen: nil,
+            prediction: nil,
+            now: nil
+
+  @type t :: %__MODULE__{
+          screen: Screen.t(),
+          prediction: Prediction.t(),
+          now: DateTime.t()
+        }
+
+  @type widget_data :: %{
+          destination: String.t(),
+          arrival_time: String.t(),
+          crowding: list(crowding_data),
+          platform_position: number,
+          front_car_direction: :left | :right,
+          now: String.t()
+        }
+
+  @type crowding_data :: %{
+          crowding_level: 1 | 2 | 3,
```

Comment:
Should these values be the level atoms from the vehicle parser? `:many_seats_available`, `:full`, etc.

Reply:
Also, we will need to add some values for closed cars and no data. I'm not 100% what that looks like in the data though. I asked [here](https://mbta.slack.com/archives/C039ARUM48H/p1692880405163159?thread_ts=1692710156.845019&cid=C039ARUM48H).

Reply:
Ah yep, this whole type was old.